code/modules/clothing/shoes/miscellaneous.dm

```diff
@@ -1,24 +1,38 @@
 /obj/item/clothing/shoes/syndigaloshes
-	desc = "A pair of brown shoes. They seem to have extra grip." //change line ~346 in code/datums/uplink_item.dm if you remove the second sentence
+	desc = "A pair of brown shoes." // the "extra grip" contraband id text is moved to examine()
 	name = "brown shoes"
 	icon_state = "brown"
 	item_state = "brown"
-	permeability_coefficient = 0.05
+	_color = "brown"
 	clothing_flags = NOSLIP
 	origin_tech = Tc_SYNDICATE + "=3"
 	var/list/clothing_choices = list()
 	actions_types = list(/datum/action/item_action/change_appearance_shoes)
 	siemens_coefficient = 0.8
-	species_fit = list(VOX_SHAPED)
+	permeability_coefficient = 0.90
+	species_fit = list(VOX_SHAPED, GREY_SHAPED, UNDEAD_SHAPED, MUSHROOM_SHAPED)
+
+// desc replacement block
+/obj/item/clothing/shoes/syndigaloshes/examine(mob/user) 
+	..()
+	if(is_holder_of(user, src)) // are the noslips on your person or on the floor? 
```

Comment:
We should probably move all this behavior up to item level and have it be its own var so we can use this for any item that should give more info when examined closely.

Reply:
Hm. Should I make a separate PR for that and make this PR DNM till it's merged?

Reply:
Also, should it work on the /obj/ level? Machinery might be able to benefit from it as well.

Reply:
The difference is that at the item level objects can be held, but it's not a bad idea to make it extensible as you say if you want to include it.

Go ahead and do it as part of this pr, then the framework is available for future prs.